torch/nn/utils/parametrize.py

```diff
@@ -282,6 +284,21 @@ def _inject_new_class(module: Module) -> None:
     """
     cls = module.__class__
 
+    def default_deepcopy(self, memo):
```

Comment:
Just a nitpick, feel free to ignore: It is not really a "default deepcopy" as the default deepcopy would fail due to the lack of serialization support.